prime-router/metadata/HL7/catchall/hl7/segments/ORC/Specimen.yml

```diff
@@ -29,9 +29,15 @@ container:
               expressionType: HL7Spec
 
 collection:
-  condition: $spm7 NOT_NULL
+  condition: $obr7 NOT_NULL || $obr8 NOT_NULL || $obr9 NOT_NULL || $obr10 NOT_NULL || $obr15 NOT_NULL
```

Comment:
I think we just chose ORC as the folder name since ORC/OBR get smashed together into a bunch of different.

Agreed. that smp7 check is wrong, but not following why OBR.15 is checked?

Reply:
OBR.15.4 is used below for collection.bodySite. I can update if the check should be specifically for that sub-field.